### PR TITLE
Add support for Python 3.9

### PR DESCRIPTION
**Pull request by MatthieuDartiailh**

### CODE REVIEW

File: examples/tutorial/hello_world/hello_world.py [hunk 1]
```diff
@@ -18,8 +18,8 @@ def main():
     view = Main(message="Hello World, from Python!")
     view.show()
 
-    # Start the application event loop
-    app.start()
+    # # Start the application event loop
+    # app.start()
```
Comment (frmdstryr) [inline examples/tutorial/hello_world/hello_world.py:22]: ?

Comment (MatthieuDartiailh) [reply examples/tutorial/hello_world/hello_world.py:22]: Need to be removed. Helped me debug the qt5 issue on Linux.

**Comment by frmdstryr** [comment]
I gave it a quick test with inkcut, everything seems to work fine. Great work!

**Comment by MatthieuDartiailh** [comment]
Thanks for testing ! Note that you should also be able to run on PySide2 now.

**Pull request merged**